src/compiler/crystal/tools/init.cr

```diff
@@ -44,17 +44,21 @@ module Crystal
     end
 
     def self.fetch_author
-      return "[your-name-here]" unless system(WHICH_GIT_COMMAND)
-      `git config --get user.name`.strip
+      default = "your-name-here"
+      return default unless system(WHICH_GIT_COMMAND)
+      user_name = `git config --get user.name`.strip
+      user_name.empty? ? default : user_name
     end
 
     def self.fetch_email
-      return "[your-email-here]" unless system(WHICH_GIT_COMMAND)
-      `git config --get user.email`.strip
+      default = "your-email-here"
+      return default unless system(WHICH_GIT_COMMAND)
+      user_email = `git config --get user.email`.strip
+      user_email.empty? ? default : user_email
     end
 
     def self.fetch_github_name
-      default = "[your-github-name]"
+      default = "your-github-name"
```

Comment:
I'd keep brackets as an easy placeholder designation.

Reply:
@Sija Yeah I wanted to keep brackets too, but then it will fail to parse `shard.yml`

```yaml
authors:
  - [your-name-here] <[your-email-here]>
```

Outputs:

```
in /home/main/blog/shard.yml: Expected SCALAR but was SEQUENCE_START at line 5, column 5

  3. 
  4. authors:
  5.   - [your-name-here] <[your-email-here]>
```

Reply:
So just quote it in the `ECR` file?